BoilerPlates/svelte-rollup/README.md

```diff
@@ -0,0 +1,214 @@
+To create a new project based on this template using [degit](https://github.com/Rich-Harris/degit):
+
+```bash
+npx degit sveltejs/template ./
+npm install
+```
+
+*Note that you will need to have [Node.js](https://nodejs.org) installed.*
+
+## Licensing
+
+SciChart.js is commercial software with a [free community license](https://scichart.com/community-licensing).
+
+- From SciChart.js v3.2 and onwards, trial licenses are not required. Instead the chart initialises with a [Community License](https://scichart.com/community-licensing)
+- For commercial licensing, follow steps from [scichart.com/licensing-scichart-js](https://scichart.com/licensing-scichart-js).
+
+## Step 1: Installing SciChart 
+If you haven't already done so, add SciChart.js to your application.
+
+```bash
+npm install scichart
+```
+
+## Step 2: Wasm file deployment
+
+SciChart.js uses WebAssembly files which must be served. The easiest way to do this is to copy the wasm files from the node_modules/scichart/\_wasm folder to your public folder.
+
+### rollup.config.js:
+
+```javascript
+export default {
+	input: 'src/main.js',
+	output: {
+		sourcemap: true,
+		format: 'iife',
+		name: 'app',
+		file: 'public/bundle.js' // change, so that everything is served at "public", not "build"
+	},
+	plugins: [
+		svelte({
+			compilerOptions: {
+				// enable run-time checks when not in production
+				dev: !production
+			}
+		}),
+
+		// Serve WASM files from SciChart into the bundle
+        // scichart2d.wasm
+		{
+			name: 'wasm',
+			generateBundle() {
+				this.emitFile({
+					type: 'asset',
+					fileName: 'scichart2d.wasm',
+					source: fs.readFileSync('node_modules/scichart/_wasm/scichart2d.wasm')
+				});
+			}
+		},
+		// scichart2d.data
+		{
+			name: 'wasm',
+			generateBundle() {
+				this.emitFile({ 
+					type: 'asset',
+					fileName: 'scichart2d.data',
+					source: fs.readFileSync('node_modules/scichart/_wasm/scichart2d.data')
+				});
+			}
+		},
+		// scichart2d.js
+		{
```

Comment:
Copying scichart2d.js is not normally required.  Is this to avoid the bundling problem?